silkworm/node/etl/mini_collector.hpp

```diff
@@ -0,0 +1,154 @@
+/*
```

Comment:
I suggest using `InMemoryCollector` instead of `MiniCollector` and renaming all stuff (also header file) accordingly.

Reply:
I have done the renaming; I also have made the collector container a template parameter and provided two implementations, one based on map and one based on vector, with different memory footprints.